_maps/basemap.dm

```diff
@@ -20,6 +21,7 @@
 //#include "map_files\generic\Space.dmm"
 //#include "map_files\generic\SpaceDock.dmm"
 #include "map_files\generic\Transit.dmm"
+//#include "map_files\generic\VR_Space.dmm"

```

Comment:
Why are you adding commented out map files?

Reply:
That was me attempting to make them their own Z-Level, but that's not really required.